doc/integrator/ngeo.rst.tmpl

```diff
@@ -94,6 +94,9 @@ The sub section is the interface name, and after that we have:
 
 * ``redirect_interface``: Interface to be redirected to if an unexpected device type is used (mobile/desktop).
 * ``do_redirect``: Directly do the redirect.
+* ``extends``: Name of the interface we want to extends the ``constants``, ``dynamic_constants``,
```

Comment:
I don't really understand. Should this be:
"Interface for which the constants, dynamic_constants, static and routes shall be extended"
?